python/tests/spark/sql/codegen/test_mlflow_registry.py

```diff
@@ -82,8 +82,10 @@ def mlflow_client(
             registered_model_name="vanilla-mlflow-no-tags",
         )
 
-    spark.conf.set("rikai.sql.ml.registry.mlflow.tracking_uri", tracking_uri)
-    return mlflow.tracking.MlflowClient(tracking_uri)
+
+@pytest.fixture(scope="session")
+def yolo_model():
+    return torch.hub.load("ultralytics/yolov5", "yolov5s")
```

Comment:
@eddyxu this is what's failing. The issue if you look at yolov5 code (https://github.com/ultralytics/yolov5) is that it's not packaged like a python module so when yolov5 imports from modules in the same repo, it fails in the unit tests.

Reply:
```
________________________________________________ ERROR at setup of test_mlflow_yolo ________________________________________________

    @pytest.fixture(scope="session")
    def yolo_model():
>       return torch.hub.load("ultralytics/yolov5", "yolov5s")

tests/spark/sql/codegen/test_mlflow_registry.py:88: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/chang/miniconda3/envs/eto/lib/python3.7/site-packages/torch/hub.py:339: in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
/home/chang/miniconda3/envs/eto/lib/python3.7/site-packages/torch/hub.py:365: in _load_local
    hub_module = import_module(MODULE_HUBCONF, hubconf_path)
/home/chang/miniconda3/envs/eto/lib/python3.7/site-packages/torch/hub.py:74: in import_module
    spec.loader.exec_module(module)
<frozen importlib._bootstrap_external>:728: in exec_module
    ???
<frozen importlib._bootstrap>:219: in _call_with_frames_removed
    ???
/home/chang/.cache/torch/hub/ultralytics_yolov5_master/hubconf.py:12: in <module>
    from models.yolo import Model
/home/chang/.cache/torch/hub/ultralytics_yolov5_master/models/yolo.py:11: in <module>
    from models.common import *
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import math
    from copy import copy
    from pathlib import Path
    
    import numpy as np
    import pandas as pd
    import requests
    import torch
    import torch.nn as nn
    from PIL import Image
    from torch.cuda import amp
    
>   from utils.datasets import letterbox
E   ModuleNotFoundError: No module named 'utils.datasets'; 'utils' is not a package

/home/chang/.cache/torch/hub/ultralytics_yolov5_master/models/common.py:15: ModuleNotFoundError
```